Dockerfile.rhel

```diff
@@ -1,10 +1,15 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.15 AS builder
+
+# Install the latest version of Delve
+RUN GOFLAGS='' go install github.com/go-delve/delve/cmd/dlv@latest
```

Comment:
We need to install delve

Reply:
Could delve be delivered via a sidecar? This way, instead of `delve ... exec` we'd run `delve attach` there`

Reply:
As long as sidecar container is privileged and SYS_PTRACE cap is enabled, I'd expect it to be working. Seems even better idea.